tests/integration/ambient/main_test.go

```diff
@@ -88,7 +88,6 @@ func TestMain(m *testing.M) {
 		Setup(istio.Setup(&i, func(ctx resource.Context, cfg *istio.Config) {
 			// can't deploy VMs without eastwest gateway
 			ctx.Settings().SkipVMs()
```

Comment:
I'm a bit confused - how does dropping this flag change things, exactly? 

```
	if cfg.EnableCNI {
		args.AppendSet("components.cni.namespace", "kube-system")
		args.AppendSet("components.cni.enabled", "true")
	}
```

how/why does this make bookinfo tests flake, exactly?

We _do_ want cni enabled for ambient?

Reply:
Just found out in my local running, the istio-cni-node pod has not been running at all.    Tried to remove this line and the istio-cni-node pod is up and running though not sure the reason

Reply:
i remember having to enabled it as some tests assume no CNI broke for me: https://github.com/istio/istio/pull/48253/commits/1b2cbdfd0e81fb2188652c9d789b16398ef1e1f9